vignettes/manual/HectorSummary.Rmd

```diff
@@ -0,0 +1,50 @@
+
+Hector contains a well-mixed global atmosphere, an ocean component, and a land component composed of vegetation, detritus, and soil. Briefly, terrestrial vegetation, detritus, and soil are linked with each other and the atmosphere by first-order differential equations. Carbon flows from vegetation to detritus to soil, losing fractional amounts to heterotrophic respiration. For more details on the terrestrial carbon cycle, please reference ["The model's terrestrial carbon cycle"](https://jgcri.github.io/hector/articles/manual/SimpleNBox.html). Note that Hector's fundamental time step is one year, but the carbon cycle can be scaled to a finer resolution if necessary.^[Hartin et al., 2015.] The ocean component consists of four boxes: two surface boxes, one high-latitude and the other low-latitude, one intermediate, and one deep box. The cold high-latitude (>55$^◦$) box makes up 15% of the ocean surface area and the warm low-latitude box (>55$^◦$) makes up the remaining 85%. The temperatures are linearly related to the global atmospheric temperature change; this temperature gradient causes a flux of carbon into the high-latitude box and a flux of carbon out of the warm low-latitude box.^[Vega-Westhoff, B., Sriver, R. L., Hartin, C. A., Wong, T. E., & Keller, K. (2019). Impacts of observational constraints related to sea level on estimates of climate sensitivity. Earth's Future, 7, 677– 690. https://doi.org/10.1029/2018EF001082]
+
+Hector can also calculate near surface global atmospheric temperature. Radiative forcing is calculated from various atmospheric greenhouse gases, aerosols, and pollutants, including halocarbons, tropospheric ozone, black and organic carbon, sulfate aerosols, methane (CH~4~), nitrous oxide (N~2~0), and stratospheric H~2~0 from methane oxidation.^[Hartin et al., 2015] 
+
+Hector v2.0 replaced the temperature component of the model with the coupled 0-D climate and 1-D ocean heat diffusion model, DOECLIM. The Hector implementation, like other past assessments using DOECLIM, has three tuneable physical parameters: equilibrium climate sensitivity to a doubling of CO~2~, vertical ocean diffusivity, and an aerosol forcing scaling factor. This addition improved vertical ocean structure and heat uptake as well as the surface temperature response to radiative forcing. Additionally, Hector V2.0 incorporates a semiempirical model based on global temperature to calculate global sea level change. This component is based on the BRICK model framework, where global sea level changes are treated as the sum of contributions from thermal expansion, glaciers and small ice caps, the Greenland and Antarctic ice sheets, and changes in land water storage. BRICK has been updated within Hector to simplify coupling.^[Vega-Westhoff et al., 2019] 
+
```

Comment:
Is it BRICK or BRICC? 😕

Reply:
https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2018EF001082

Should be BRICK :)

Thanks for your comments!